JASP-Engine/JASP/R/commonAnovaBayesian.R

```diff
@@ -258,7 +258,7 @@
           iterations   = bfIterations))
         if (isTryError(bf)) {
           modelName <- .unvf(strsplit(.BANOVAas.character.formula(model.list[[m]]), "~ ")[[1L]][2L])
-          .quitAnalysis(sprintf("Bayes factor could not be computed for model: %s.\nThe error message was: %s.",
+          .quitAnalysis(gettextf("Bayes factor could not be computed for model: %s.\nThe error message was: %s.",
```

Comment:
gettextf("Bayes factor could not be computed for model: %1$s.\nThe error message was: %2$s."